packages/svelte2tsx/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "svelte2tsx",
-    "version": "0.7.25",
+    "version": "0.7.26",
```

It's actually going to be this
```suggestion
    "version": "0.7.35",
```
It's confusing, because we bump those by hand only from time to time, but at deploy time it is basically ignored in favor of what the release name is.